PythonRegression/tests/features/machine1/1_api_tests.feature

```diff
@@ -38,56 +53,174 @@ Feature: Test API calls on Machine 1
+        Given "checkConsistency" is called on "nodeA" with:
+        |keys           |values             |type           |
+        |tails          |TEST_HASH          |staticList     |
+
+        Then a response with the following is returned:
+        |keys                               |
+        |duration                           |
+        |info                               |
+        |state                              |
+
+
+    #Values can be found in util/static_vals.py
+    Scenario: GetInclusionStates is called
+	    Given "getInclusionStates" is called on "nodeA" with:
+        |keys           |values             |type               |
+        |transactions   |TEST_HASH          |staticList         |
+        |tips           |TEST_TIP_LIST      |staticValue        |
+
+        Then a response with the following is returned:
+        |keys                               |
+        |duration                           |
+        |states                             |
+
+
+    #Address can be found in util/static_vals.py
+    Scenario: GetBalances is called
+        Given "getBalances" is called on "nodeA" with:
+        |keys       |values                 |type               |
+        |addresses  |TEST_EMPTY_ADDRESS     |staticList         |
+        |threshold  |100                    |int                |
+
+
+        Then a response with the following is returned:
+        |keys                               |
+        |balances                           |
+        |duration                           |
+        |milestoneIndex                     |
+        |references                         |
+
+
+    Scenario: Interrupt attach to tangle
+        Given "attachToTangle" is called on one thread in "nodeA" with:
+        |keys                   |values                         |type           |
+        |trytes                 |EMPTY_TRANSACTION_TRYTES       |staticList     |
+        |trunk_transaction      |TEST_HASH                      |staticValue    |
+        |branch_transaction     |TEST_HASH                      |staticValue    |
+        |min_weight_magnitude   |50                             |int            |
+
+        And "interruptAttachingToTangle" is called on one thread in "nodeA" with:
+        |keys                   |values                         |type           |
+
+        When all api call threads are joined
+        #The only active thread should be the main testing thread
+        Then while checking for running threads, there should be "1" running
+
+
+
+
+
+
+    Scenario: WereAddressesSpentFrom is called
+        Given "wereAddressesSpentFrom" is called on "nodeA" with:
+        |keys       |values                 |type               |
+        |addresses  |TEST_EMPTY_ADDRESS     |staticList         |
+
+    	Then a response with the following is returned:
+    	|keys                               |
+    	|duration                           |
+    	|states                             |
+
```

Comment:
Since we have a static db, maybe you can check the returned values and not just the keys